libc/hdr/CMakeLists.txt

```diff
@@ -41,6 +41,24 @@ add_proxy_header_library(
     libc.include.fenv
 )
 
+add_proxy_header_library(
+  fenv_t
```

Comment:
types should go in libc/hdrs/types/

Reply:
ok thanks

Reply:
and do I need to change the corresponding `libc.hdr.fenv_t` to `libc.hdr.types.fenv_t` in the CMakeList.txt?

Reply:
yeah

Reply:
thanks

Reply:
Ok I have addressed your comments